azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeTelecomOptions.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+/**
+ * CallCompositeTelecomOptions {@link CallCompositeTelecomOptions}.
+ */
+public class CallCompositeTelecomOptions {
```

Comment:
Do we need this class if it just wraps one field?

Reply:
As per discussion we will fix for GA.